### DISCUSSION TITLE
New version of Apps docs is live ✨

### DISCUSSION THREAD

**Discussion by peelar** [saleor/saleor#11246] [Announcements]
Hello everyone 👋 

We are working hard on making the experience of building Saleor apps seamless. Part of that effort is documenting all the great features our DevTools Team has built.

And as they are working extremely hard, we recently had quite a lot to cover in the documentation. We used this opportunity to slightly revamp the intro "Apps" page, restructure the content, and add some new ones.

The new structure looks like that:
- 📏 Architecture (Communication, APL, Requirements for building an app from scratch)
- 🔨 Developing (Tunneling, SDK, App Template, App Examples)
- 🔃 Webhooks

We are definitely not slowing down. Apps are very important to us, so expect to see more buzz around them in the near future 🔉. If you have any feedback regarding the docs, we would be very grateful if you shared it.

[Start learning about apps now ▶️](https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts)

**Comment by barrybhattarai** [comment]
I'll be glad if we could bring Saleor to Flutter officially. This would be a milestone.

**Reply by peelar** [reply]
Hi @barrybhattarai 👋 

I personally watch Flutter from afar with admiration, I am definitely a fan.

What would "bringing Saleor to Flutter officially" mean to you?

**Reply by peelar** [reply]
BTW. [Here](https://github.com/saleor/example-flutter) is an example of working with Saleor in Flutter.
